arm-relay/2016-07-01/swagger/relay.json

```diff
@@ -0,0 +1,1930 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Relay API",
+    "description": "Use these API to manage Azure Relay resources through Azure Resources Manager.",
+    "version": "2016-07-01",
+    "x-ms-code-generation-settings": { "name": "RelayManagementClient" }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Relay/Namespace": {
```

Comment:
These urls look inconsistent, the path below are of the format "/subscriptions/{subscriptionId}/providers/Microsoft.Relay/Namespaces" while this ends with "/Namespace", sticking to something consistent would mean the operationIds would be named as ```Namespaces_list``` which would be consistent with the rest of the urls and their operationIds